lib/evaluation/core/selectResults.ts

```diff
@@ -19,7 +19,7 @@ export interface SelectResults {
   /**
    * All matched/selectable credentials
    */
-  verifiableCredential?: IVerifiableCredential[];
+  verifiableCredential?: OriginalVerifiableCredential[];
```

Comment:
Although this a good change: Is this exposed to the outside world? and then is this compatible with the current interface?

Reply:
type of this was always OriginalVerifiableCredential. there were some `as` keywords for being able to pass it as an `IVerifiableCredential`